pulp_ansible/app/models.py

```diff
@@ -268,6 +268,15 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
 
+def _get_last_sync_task(pk):
+    sync_task = Task.objects.filter(name__contains="sync", reserved_resources_record__icontains=pk)
+
+    if sync_task:
+        return sync_task.order_by("-pulp_created").first()
+
+    return None
```

Comment:
```suggestion
def _get_last_sync_task(pk):
    sync_tasks = Task.objects.filter(name__contains="sync", reserved_resources_record__icontains=pk)
    return sync_tasks.order_by("-pulp_created").first()
```

`first` has this logic builtin.

Reply:
thanks, I thought this was throwing an error, but it was actually using `reserved_resources_record__contains` instead of  `reserved_resources_record__icontains`

```
pulp_1   | pulp [4f173387cd7a494c84129a1f7cde3c17]: django.request:ERROR: Internal Server Error: /api/automation-hub/pulp/api/v3/repositories/ansible/ansible/fddb35fa-b626-401b-a3a3-7f93d2d24dce/
pulp_1   | Traceback (most recent call last):
pulp_1   |   File "/usr/local/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
pulp_1   |     return self.cursor.execute(sql, params)
pulp_1   | psycopg2.errors.CannotCoerce: cannot cast type uuid to text[]
pulp_1   | LINE 1: ...d" @> 'fddb35fa-b626-401b-a3a3-7f93d2d24dce'::uuid::text[]) ...
pulp_1   |                                                              ^
```